### ISSUE TITLE
Installfest breaks on El Capitan Beta

### ISSUE THREAD

**Issue by h4w5** [GA-WDI/installfest#63]
@jadencarver @CentroDL : [OS check](https://github.com/ga-instructors/installfest_script/blob/master/builds/mac#L166) fails on [10.11](http://www.macrumors.com/roundup/os-x-10-11/). It also begins a full XCode download and install, instead of just XCode command line tools.

**Comment by scribbleboyed** [comment]
Hi, Phil here again. The failures cascade down the install:

-   the command line tools check fails
-   the diskutil fails since there is no longer a repairPermissions verb

    ```
    diskutil: did not recognize verb "repairPermissions"; type "diskutil" for a list
    ```

**Comment by CentroDL** [comment]
Glad to help.

1. Which branch is the script running off of? 
2. What info do we have on the target laptop? 

As far as we know the Pluto Script is working so far, we just ran is yesterday for our class.

**Comment by h4w5** [comment]
@CentroDL They were running off of master. Sorry I couldn't more info (was running a different class at the moment), but the OS version was 10.11.

**Comment by h4w5** [comment]
Fixed with 8bf062b.
